src/imagewriter.h

```diff
@@ -209,7 +222,7 @@ protected slots:
 protected:
     QUrl _src, _repo;
     QString _dst, _cacheFileName, _parentCategory, _osName, _currentLang, _currentLangcode, _currentKeyboard;
-    QByteArray _expectedHash, _cachedFileHash, _cmdline, _config, _firstrun, _cloudinit, _cloudinitNetwork, _initFormat;
+    QByteArray _expectedHash, _cachedFileHash, _cacheFileHash, _cmdline, _config, _firstrun, _cloudinit, _cloudinitNetwork, _initFormat;
```

Comment:
One might argue that having two member-variables with only a one-character difference (`_cachedFileHash` and `_cacheFileHash`) might make them easy to confuse?  :laughing: